README.md

```diff
@@ -71,7 +71,7 @@ Code changes are viewable on GitHub [Link](https://github.com/CrocSwap/ambient-t
 ### version 1.6
 Summary:
 * Added the ability to switch to the Scroll network and a test network (Görli) using the network selector in the page header
-* Added an external link to [Canto](https://beta.canto.io/lp) in the network selector
+* Added an external link to [Canto](https://canto.io/lp) in the network selector
```

Comment:
Version notes are to provide an historical record of changes to the codebase. This line should not be updated as it was accurate as of the version 1.6 release.